cmd/gpu_plugin/README.md

```diff
@@ -224,9 +224,32 @@ master
 
 ## Testing and Demos
 
-We can test the plugin is working by deploying an OpenCL image and running `clinfo`.
-The sample [intel-opencl-icd](../../demo/intel-opencl-icd/) OpenCL image can be built
-using `make intel-opencl-icd` and must be made available in the cluster.
+We can test the plugin is working by deploying an [OpenCL image](../../demo/intel-opencl-icd/) that is running `clinfo`.
+
+1. Make the image available to the cluster:
+
+    Build image:
+
+    ```bash
+    $ make intel-opencl-icd
+    ```
+
+    Tag and push the `intel-opencl-icd` image to a repository available in the cluster. And modify the `intelgpu-job.yaml`'s image location accordingly:
+
+    ```bash
+    $ docker tag intel/intel-opencl-icd:devel <repository>/intel/intel-opencl-icd:latest
+    $ docker push <repository>/intel/intel-opencl-icd:latest
+    $ vi ${INTEL_DEVICE_PLUGINS_SRC}/demo/intelgpu-job.yaml
+    ```
+
+    If you are running the demo on a single node cluster, you can push the image to the CRI's repository. For example, docker to containerd:
+
+    ```bash
+    $ IMAGE_NAME=opencl-icd.tar
+    $ docker save -o $IMAGE_NAME intel/intel-opencl-icd:devel
+    $ ctr -n=k8s.io images import $IMAGE_NAME
+    $ rm $IMAGE_NAME
+    ```
```

Comment:
I think the description is too technical.  Maybe something like:
> If you are running the demo on a single node cluster, and do not have your own registry, you can add image to node image cache instead. For example, to import docker image to containerd cache:

(I guess whether not having image in a registry actually works, depends on pod image policy.  With "always" one it would not work...)

PS.  IMHO description of how to use images without a registry (on single-node system where it's not too much of trouble) would better in some common device plugins document.  GPU plugin README should just link that common doc.